Application/src/visuals/Message.java

```diff
@@ -1,25 +1,27 @@
 package visuals;
 
 import javafx.scene.canvas.GraphicsContext;
-import javafx.scene.paint.Color;
 import javafx.scene.image.Image;
-import visuals.handlers.Animation;
 
 /**
  * Class for creating the messages to pass between "classes".
+ *
  * @author Sebastian Fransson
- * @version 1.0
+ * -collaborator Rashad Kamsheh
+ * @version 1.1
  */
-public class Message implements Renderable{
+public class Message implements Renderable {
     private String name;
-    private Coordinates coordinates , node1, node2; // The coordinates of the nodes that the message is supposed to pass between.
+    private Coordinates coordinates, node1, node2; // The coordinates of the nodes that the message is supposed to pass between.
```

Comment:
Why is this changed?

Reply:
All of those indentation changes happend because of i used the fomatting command in Intelj
Would you like them to be changed back?